module/zfs/dsl_scan.c

```diff
@@ -236,24 +236,37 @@ typedef enum {
  */
 typedef struct scan_io {
 	/* fields from blkptr_t */
-	uint64_t		sio_offset;
 	uint64_t		sio_blk_prop;
 	uint64_t		sio_phys_birth;
 	uint64_t		sio_birth;
 	zio_cksum_t		sio_cksum;
```

Comment:
I talked to Brian about this a little bit. I think the savings are significant, especially because the size of the vdev queues is bounded by how much memory these structs take up. The smaller these structs are the more sequential IO we can queue up.

I just realized that I did not update the accounting code to account for the variable number of DVAs at the end of the struct. I will do that now.